README.md

```diff
@@ -19,19 +19,55 @@ Install the package:
 composer require andreaselia/laravel-api-to-postman
 ```
 
-With auto-discovery you don't need to do anything else.
+Publish the config file:
+
+```bash
+php artisan vendor:publish --provider="AndreasElia\PostmanGenerator\PostmanGeneratorServiceProvider" --tag="config"
+```
+
+## Configuration
+
+You can modify the `api-postman.php` config values:
+
+- `structured` - If you want folders to be generated based on route names.
+- `base_url` - The base URL for all of your endpoints.
+- `auth_middleware` - The middleware which wraps your authenticated API routes.
 
 ## Usage
 
+The output of the command being ran is your storage/app directory.
+
 To use the command simply run:
 
 ```bash
 php artisan export:postman
 ```
 
-- `--structured` generates routes in folders based on their namespace
-- `--bearer=<token>` generates a token variable in Postman for the specified token
-- `--base-url=<base_url>` defaults to https://api.example.com/ unless specified
+The following usage will generate routes with the bearer token specified.
+
+```bash
+php artisan export:postman --bearer=1|XXNKXXqJjfzG8XXSvXX1Q4pxxnkXmp8tT8TXXKXX
+```
+
+## Examples
+
+This is with the default configuration and a bearer token passed in:
+
+```bash
+php artisan export:postman --bearer=123456789
+```
+
+Given the following `api.php` routes:
+
+https://gist.github.com/AndreasElia/e4dbfc751e577add2c3eaaccaba5ad11
+
+The output would be the following JSON which can be imported into Postman as a collection:
+
+https://gist.github.com/AndreasElia/ffe8d7def3f108c452927e422d29fa9d
+
+If you specify `structured` as `true` in the configuration file, the output will be:
+
+https://gist.github.com/AndreasElia/7c7aee08dea40b6ff0b8ef30547ab721
```

Comment:
Thoughts on moving these to an `examples` folder in the source? 🤔

Reply:
Sorted